lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { repositories: { LhcPeriodStatisticsRepository } } = require('../../../database');
+const { dataSource } = require('../../../database/DataSource.js');
+const { ApiConfig } = require('../../../config');
+const { lhcPeriodStatisticsAdapter } = require('../../../database/adapters');
+
+/**
+ * @typedef LhcPeriodIdentifier object to uniquely identify a lhc period
```

Comment:
`LhcPeriodStatisticsIdentifier`, it uniquely identify a LHC period STATISTICS
Maybe somewhere you should explain what is LhcPeriodStatistics compared to plain LhcPeriod, maybe in the model (or in `data-model.md`)